package.json

```diff
@@ -87,6 +87,9 @@
     "lint": "npm run lint:css && npm run lint:js",
     "lint:js": "eslint .",
     "lint:css": "stylelint public/css/",
+    "fix": "npm run fix:css && npm run fix:js",
```

Comment:
Good idea!  Keep in mind you can also run this fix automatically as part of format-on-save VS Code action.  Let me know if you'd like me to help you set that up.